strr-web/lang/en.json

```diff
@@ -233,6 +234,12 @@
       "haveAnotherProperty": "Have another rental property?",
       "startNewApplication": "Start a new application"
     },
+    "propertyManager": {
+      "title": "Step 1 - Property Manager Information",
+      "subtitle": "If there is a property manager business for this short-term rental unit, provide the contact information below.",
+      "primary": "Property Manager Information"
+    },
+    "propertyManagerForm": {},
     "contact": {
       "title": "Step 1 - Contact Information",
```

Comment:
Should this be Step 2 now, since the Property Manger Information is Step 1?

Reply:
Fixed